components/campaign-analytics/Card.tsx

```diff
@@ -14,23 +14,23 @@ const Card = ({ title, amount = "0", analytics = "+ 0%" }: CardProps) => {
 	const [item, setItem] = useState("All item");
 
 	return (
-		<div className="flex h-[180px]  w-full flex-col justify-center gap-3 rounded-xl border-CampaignCardBorderWidth border-placeholder bg-white p-4 text-sm text-abeg-text text-opacity-80 md:text-base lg:gap-4">
+		<div className="flex h-[180px] w-full flex-col justify-center gap-3 rounded-xl border-CampaignCardBorderWidth border-placeholder bg-white p-4 text-sm md:text-base text-abeg-text text-opacity-80 lg:gap-4">
```

Comment:
why are you specifying a definite height here?

Reply:
also use gap for the spacing and don't add margin top to the children element. let there be one source of truth